CASC/PortfolioMode.cpp

```diff
@@ -582,7 +582,7 @@ void PortfolioMode::runSlice(Options& strategyOpt)
   if(outputResult) { // this get only true for the first child to find a proof
     ASS(!resultValue);
 
-    if (outputAllowed() && (Lib::env.options && Lib::env.options->multicore() != 1)) {
```

Comment:
We use `env.options` above and below here, so we know it isn't `nullptr`. `Lib::env` and `env` are the same, I think.